.github/workflows/reuse.yaml

```diff
@@ -102,27 +102,32 @@ jobs:
         id: check-branch
         run: |
           if [[ ${{ steps.extract_branch.outputs.branch }} =~ ^release-[0-9]+\.[0-9]$ ]]; then
-              echo ::set-output name=match::true
-          fi
+                echo ::set-output name=match::true
+            fi
```

Comment:
Why are you changing the indentation here ?

Reply:
my bad 🤦🏻‍♂️